home.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Simple Server >> Simple Syrup, Cheers!</title>
+</head>
+<body>
+  <h1>The headeriest of headers!</h1>
+  <p>The textiest text you ever saw.</p>
```

Comment:
Catchy! You could go into copywriting if this engineering stuff doesn't work out.